assets/prometheus-k8s/service-account.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
-automountServiceAccountToken: false
+automountServiceAccountToken: true
```

Comment:
I think that we should override this in the CMO jsonnet since it worked fine for us (IIUC it's needed when using a version of Prometheus operator that doesn't set `automountServiceAccountToken: true` at the statefulset level).

Reply:
ok, will overide this option at CMO level.